content/vgg-classifier-tutorial.md

```diff
@@ -0,0 +1,320 @@
+Title: Keras, Make your own image classifier
+Date: 2016-03-30
+Category: Tutorial
+Author: Gregory Senay
+
+
+This tutorial explains how to create image classifier, from an ImageNet pre-trained model, your own model on your own dataset.
+
+Prepare your dataset
+-----
+Here, I will explain how to simply prepare your image dataset following an existing simple one: Caltech101.
+Caltech101 is a dataset of 101 categories (crocodile, camera, plane, soccer ball...) with 40 to 800 images per category, the size of image is bout 200x300.
+To prepare your data, you have to follow this directory architecture:
+```
+  * Main Directory # Name of you dataset
+    * Class1Name
+      * Image1Class1
+      * Image2Class1
+      * Image3Class1
+      * ...
+    * Class2Name
+      * Image1Class1
+      * Image2Class1
+      * ...
+    * Class3Name
+      * Image1Class1
+      * Image2Class1
+      * ...
+    * Class4Name
+    * ...
+
+```
+
+In the case of Caltech101, (available here: http://www.vision.caltech.edu/Image_Datasets/Caltech101/101_ObjectCategories.tar.gz), once you extract the archive with ``tar xvf 101_ObjectCategories.tar.gz)`` you have this directory architecture:
```

Comment:
Download link should be introduced in a separate line: "The Caltech101 data is available for download [from this link]." Please use Markdown link syntax to specify hyperlinks.